shared/profile/edit-profile/index.js

```diff
@@ -35,6 +35,7 @@ export default compose(
     bioLengthLeft: props.bio ? maxProfileBioChars - props.bio.length : maxProfileBioChars,
   })),
   withHandlers({
+    onCancel: ({onBack}) => () => onBack(),
```

Comment:
oh you want both. k well you can use withHandlers or just add it to mapDisatpchToProps